src/content/1.7/modules/concepts/controllers/admin-controllers/_index.md

```diff
@@ -102,3 +103,31 @@ Since Symfony 4.1 the bundle notation is going to be deprecated: https://symfony
 {{% /notice %}}
 
 The Controller in the previous example will be available if you browse `/admin-dev/modules/your-module/demo`.
+
+
+## How to map an action of your controller to a URI
```

Comment:
Looks like I duplicated the content above 😭 bad copy-paste

Reply:
:thinking: From where :thinking:

Reply:
Ohhhh